DS-Session2-Question2/src/meta/utility/CircularQueue.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * @author Abhishek Bishnoi
+ * @since Jul 30, 2019
+ * Assumption all queue values are non zero values
+ */
+package meta.utility;
+
+public class CircularQueue implements Queue {
```

Comment:
As per my understanding, the assignment is not about implementing a circular queue. Its about implementing a queue using array and addressing the problem of unused space if we create a circular queue using the queue.